CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 ### Changed
+- Change the timezone name on GUI.[#568](https://github.com/rokwire/events-manager/issues/568)
```

Comment:
@bingzhang, could you please add this to the bottom of the `Changed` section? Keep-a-Changelog format follows a top-down reading order when it comes to change entries within a section as well as sections. Thanks.